assets/js/script.js

```diff
@@ -133,87 +163,174 @@ function timerStart() {
+ */
+
+function victory() {
+    $('#victoryWindow').modal('show');
+}
+
+/*
+ * Wrong answer function
+ */
+
 function wrongAnswer() {
-    alert('Sorry, wrong answer\nCorrect answer is ' + selection.cor);
     $('#' + selection.cor).addClass('correct-answer');
     gameOver();
 }
 
-// Game over function
+/*
+ * Game over function, used when player clicks on wrong answer
+ */
 
 function gameOver() {
-    step = step - 1;
-    resetTimer();
-    saveScores();
-    $('#countdown').hide();
-    $('#step').text(step + 1);
-    $('#prizeWon').text(prize[step]);
-    $('#gameOverWindow').modal('show');
+    if (step === 0) {
+        resetTimer();
+        $('#countdown').hide();
+        $('#step').text(`You've failed to answer the first question`);
+        $('#prizeWon').text(`You didn't win any money :-(`);
+        $('#gameOverWindow').modal('show');
+    } else {
+        resetTimer();
+        saveScores();
+        $('#countdown').hide();
+        $('#step').text(`You've reached question ${step}`);
+        $('#prizeWon').text(`Your prize is ${prize[step-1]}`);
+        $('#gameOverWindow').modal('show');
+    }
 }
 
-function playAgain() {
+/* 
+ * Function called when game fails either by out of time event or by selecting the wrong question
+ */
+
+$('.playAgainButton').click(function() {
     resetTimer();
     setInterval(function() {
         location.reload();
-    }, 2000);
-}
+    }, 700);
+});
 
-// Time Out function
+/*
+ * Time Out function, used when player doesn't click on any answer
+ */
 
 function timeOut() {
-    $('#countdown').hide();
+    $('#countdown').html('<i class="fas fa-hourglass-half"></i> 0');
     $('#timeOutWindow').modal('show');
 }
 
-// Save scores function
+/* 
+ * Save scores function, only saving results with more than 3 questions answered correctly
+ */
 
 function saveScores() {
-    var score = {
-        step: step,
-        score: prize[step],
-        player: player,
-    };
-    highScores.push(score);
-    highScores.sort((a, b) => b.step - a.step);
-    highScores.splice(10);
+    if (step >= 3) {
+        let score = {
+            step: step,
+            score: prize[step],
+            player: player,
+        };
+        highScores.push(score);
+        highScores.sort((a, b) => b.step - a.step);
+        highScores.splice(10);
+
+        localStorage.setItem('highScores', JSON.stringify(highScores));
+    }
+}
+
+/* 
+ * Function to reset all buttons after successful answer
+ */
+
+function showAllButtons() {
+    $('.answers-buttons-rows').removeClass('hide');
+    if (chances !== 0) {
+        $('#chancesButton').removeClass('disabled');
+    } else {
+        $('#chancesButton').text('All chances used').addClass('disabled').prop("disabled", true);
+    }
+}
+
+
+/* 
+ * Function to hide 2 incorrect answers when Chance button is clicked
+ */
+
+$('#chancesButton').click(function() {
+    chances = chances - 1;
+    if (selection.cor === 'answer_a' || selection.cor == 'answer_b') {
```

Comment:
rewrite this line more elegantly. Put the answer values into an array and check if the array contains selection.cor.